vcs/core.go

```diff
@@ -0,0 +1,156 @@
+// Package vcs is a simplified/stripped down version of go/internal/get/vcs that
+// is aimed at the simplier temporary git clone needed for OpenFaaS template fetch.
+package vcs
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+// following utils were copied and modified from
+// https://github.com/golang/go/blob/master/src/cmd/go/internal/get/vcs.go
```

Comment:
My maintainer's hat is going on now. What is the licensing implication of copying this code?